tap_googleads/dynamic_query_stream.py

```diff
@@ -18,9 +20,37 @@
 class DynamicQueryStream(ReportsStream):
     """Define dynamic query stream class."""
 
-    records_jsonpath = "$.results[*]"
     add_date_filter_to_query = False
 
+    def get_new_paginator(self):
+        return SinglePagePaginator()
+
+    def _request(self, prepared_request, context):
+        response = self.requests_session.send(
+            prepared_request,
+            stream=True,
+            timeout=self.timeout,
+            allow_redirects=self.allow_redirects,
+        )
+        self._write_request_duration_log(
+            endpoint=self.path,
+            response=response,
+            context=context,
+            extra_tags={"url": prepared_request.path_url}
+            if self._LOG_REQUEST_METRIC_URLS
+            else None,
+        )
```

Comment:
I guess we can get rid of this logic copied over from the super implementation, since we aren't currently processing request duration metrics. This would also help reduce the amount of logs quite considerably.

Reply:
https://github.com/Matatika/tap-googleads/pull/111/commits/4700aefaa099e27b33d71bc532fd82f5e7af8a67